assets/js/googlesitekit-api.js

```diff
@@ -0,0 +1,10 @@
+/**
+ * External dependencies
+ */
+import * as API from 'assets/js/googlesitekit/api';
+
+if ( typeof window.googlesitekit === 'undefined' ) {
+	throw new Error( '`googlesitekit` is undefined. You need to import `googlesitekit` to use the `googlesitekit-api` library.' );
+}
```

Comment:
We should prepare this script to be able to stand alone (after the Webpack bundle enhancements from #993, this file should not have _any_ dependencies). In other words, let's add `window.googlesitekit = window.googlesitekit || {};` here instead of throwing an error. If `window.googlesitekit` doesn't exist here yet, there is no actual problem because this script does not rely on it, so we shouldn't throw an error.

Reply:
I think we may want to revisit that behaviour in the future, but fair enough. I guess theres' no harm in doing that for now. 👍